helm/templates/deployer/deployerRoleBinding.yaml

```diff
@@ -0,0 +1,14 @@
+kind: RoleBinding
```

Comment:
If we're wrapping the `ServiceAccount` in a toggle, let's make sure to wrap the RoleBinding, eg `{{ if .Values.deployerServiceAccount.enabled }}`